### ISSUE TITLE
Research table UI should mimick inventory UI

### ISSUE THREAD

**Issue by Cwpute** [EmptyStar/researcher#6]
The inventory shown by the research table is different from the inventory in Mineclonia, which is a bit confusing when you're used to your organized mess, mess with by the UI. Which is not the case when you switch to the Research tab in your inventory.
![2024-12-12_13-16_1](https://github.com/user-attachments/assets/3d42321c-86bb-4a5e-91bd-c9fdce640ce7)
![2024-12-12_13-16](https://github.com/user-attachments/assets/cd617e8d-6a6c-4ee1-9dc0-7316c9466f27)
![2024-12-12_13-17](https://github.com/user-attachments/assets/59b22772-8ac6-4fff-b759-222682055e0e)

**Comment by EmptyStar** [comment]
I agree, this was definitely an oversight on my part. I'll see if I can build a separate research table UI for Mineclonia/VoxelLibre that renders the inventory properly.

**Comment by Cwpute** [comment]
Yes, i noticed later, this was due to me using Mineclonia, and the UI being adapted to VoxelLibre first !
Thanks for being willing to take some time to do that :)
